CS3230Project/DAL/Accounts/AccountsDal.cs

```diff
@@ -47,7 +47,7 @@ public static bool Login(string userName, string password)
 
             using var connection = new MySqlConnection(Connection.ConnectionString);
             connection.Open();
-            const string query = "select firstName, lastName, nurses.employeeID, userName " +
+            const string query = "select *" +
```

Comment:
We should not be using '*' to select the rows, that was said by Yang in class. This should be reverted to specify all the values for nurse. I think we have this issue in more DAL files though so maybe we should think about making a separate issue for this later.

Reply:
It will be updated.

Reply:
I will take a look at this and try to update the query. I agree, we probably should plan to review all of our queries.